### ISSUE TITLE
Game crash,Version:2.1.2

### ISSUE THREAD

**Issue by wanli1028** [RyMaxim/cotw-mod-builder#2]
Game always crash when useing or click the gun after modify rifle's gauge.Please check this function of the Mod,Thanks a lot!

**Comment by wanli1028** [comment]
![屏幕截图 2024-08-04 215416](https://github.com/user-attachments/assets/41471e7a-940a-4a12-85ef-4d3a9135878a)
![354904564-9648f7b9-f00c-438e-8688-306386022bce](https://github.com/user-attachments/assets/b1d97615-7f4e-4591-a2a0-f1415a33c1ab)
![354904576-21bffcdd-6c01-4f32-b274-9fedf3b96cb7](https://github.com/user-attachments/assets/1a3cd599-b1fd-4f92-933f-0965f37440a4)
![354904567-fcaba8fa-3e99-4d51-817f-3e5728af86c7](https://github.com/user-attachments/assets/69bad853-654f-4a59-8119-1c02cce6d532)
![354904577-f0bfb66c-a490-4ac1-be80-dc12ffd2f842](https://github.com/user-attachments/assets/d742f467-b675-48cb-9f06-22dd68ef44f6)

**Comment by wanli1028** [comment]
![屏幕截图 2024-08-04 215416](https://github.com/user-attachments/assets/dcba2eb7-2146-4a86-945f-90caa2499089)
![354904564-9648f7b9-f00c-438e-8688-306386022bce](https://github.com/user-attachments/assets/38cb205b-61d7-4b24-9a5e-bf15a653b779)
![354904567-fcaba8fa-3e99-4d51-817f-3e5728af86c7](https://github.com/user-attachments/assets/065828a8-3cfa-4fcf-b1c1-e3f1ee7e1663)
![354904576-21bffcdd-6c01-4f32-b274-9fedf3b96cb7](https://github.com/user-attachments/assets/b7142e5a-6719-4eaa-aef6-a8f0a2239952)
![354904577-f0bfb66c-a490-4ac1-be80-dc12ffd2f842](https://github.com/user-attachments/assets/f06bc920-a576-46c5-b1f9-2f80195e4885)

**Comment by RyMaxim** [comment]
Can you check your Steam launch options and confirm the settings? I've been testing the Modify Weapons mod tonight while making some changes and it has been working with the following launch options:
```
--vfs-fs dropzone --vfs-archive patch_win64 --vfs-archive archives_win64 --vfs-fs .
```
I think the `--vfs-archive patch_win64` part is crucial and is currently missing from the recommended launch options on the NexusMods page.

**Comment by wanli1028** [comment]
I'm useing the Cpypasta'old launch options: --vfs-fs dropzone --vfs-archive patch_win64 --vfs-archive archives_win64 --vfs-archive dlc_win64 --vfs-fs.

**Comment by RyMaxim** [comment]
I just pushed v2.1.3 that should fix this. I missed the updated weapons files from the July patch when I released v2.1.2 and have included them now. You should be able to play with v2.1.3 using the full, proper launch options again.

I had also been working on proper names for weapons in `Modify Weapons` and `Increase Magazine Capacity` and those updates went out with the new release. If you have any saved presets using those two mods be sure to update them and save the new preset!
